docs/create-an-account.md

```diff
@@ -37,11 +36,5 @@ Simply go to the [DNAnexus log in page](https://platform.dnanexus.com/login?clie
 !!! note 
     On step 4, you must enter the billing contact's name, physical address, email address and phone number. You do not need to enter any credit card information. Once the billing contact has verified the account upgrade request, your account will be credited $50.
 
-### Lab billing setup
```

Comment:
Do we really want to remove this @mtreadwa? I feel like there is the potential for St. Jude employees to stumble across this and have this question.

Reply:
Kirby suggested a while back (and I agree after discussing) that the Create an Account page should just contain information for the external users. And that all account information relevant to internal users should live on the SJ intranet pages. So as to keep the two processes very distinct and not create confusion.  There's a note at the top of this page directing internal users to the intranet link.